google_redis/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "authorized_network" {
+  description = "The network name of the shared VPC"
+  type        = string
+}
+
+variable "custom_name" {
+  default     = ""
+  description = "Use this field to set a custom name for the redis instance"
+  type        = string
+}
+
+variable "application" {
+  description = "Application e.g., bouncer."
+  type        = string
+}
+
+variable "environment" {
+  description = "Environment e.g., stage."
+  type        = string
+}
+
+variable "realm" {
+  description = "Realm e.g., nonprod."
+  type        = string
+}
+
+variable "project_id" {
```

Comment:
I wonder if we should try to remove this variable, and switch to using the provider instead to determine what project it's installed in

Reply:
good point, let me see if I can make it flexible enough for both

Reply:
@jbuck I solved for this here: https://github.com/mozilla/terraform-modules/pull/13/commits/f4a515fb13ab2409488fd257d4b17d00776e2206

It looks like I can make the `project_id` variable's default value as `null` which will then use the provider block or env var etc. If it's not `null` it will use whatever was provided with that variable.

Reply:
Nice, that's elegant